backend/uclapi/dashboard/migrations/0006_auto_20170516_2014.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
```

Comment:
I may need to do a merge migration to get in line with master, but I don't want to do that until the last moment if possible.